pkg/reconciler/pipelinerun/resources/pipelineref.go

```diff
@@ -63,28 +61,14 @@ func GetPipelineFunc(ctx context.Context, k8s kubernetes.Interface, tekton clien
 	}
 
 	switch {
-	case pr != nil && pr.Bundle != "":
```

Comment:
Oh, it's time for this to go away!
This is supported through v1 conversion logic, right?

Reply:
yes they are handled athttps://github.com/tektoncd/pipeline/blob/eb67e5a2d2df6255b77d34b3fe077722d6406ade/pkg/apis/pipeline/v1beta1/pipelineref_conversion.go#L47 :)